src/ir/term/mod.rs

```diff
@@ -2093,5 +2093,29 @@ impl Computation {
     }
 }
 
+#[derive(Clone, Debug, Default)]
+/// A map of IR computations.
+pub struct Computations {
+    /// A map of function name --> function computation
+    pub cs: BTreeMap<String, Computation>,
```

Comment:
Can you document the reason for using a treemap instead of a hashmap.

Reply:
Also, Let's use a longer name for this field. Especially since `cs` means "children" elsewhere.

Reply:
> Can you document the reason for using a treemap instead of a hashmap.

No reason in particular, changed to FxHashMap!